src/blocks/_catalog.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap-reverse;
+  flex-shrink: 0;
+  width: $catalog-width;
+  padding: 50px 40px ;
+
+  &--font_400 {
+    font-size: 10px;
+    font-weight: 400;
+  }
```

Comment:
make that a mixin, or a placeholder class

Reply:
Nice suggestion ,  modificator has been sublstituted by placeholder accordingly.